include/revng/AutoEnforcer/AutoEnforcerLibraryRegistry.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringMap.h"
+
+#include "revng/AutoEnforcer/PipelineLoader.h"
+
+namespace AutoEnforcer {
+
+class AutoEnforcerLibraryRegistry {
```

Comment:
Let's not prefix class names with the name of the containing namespace.

Reply:
is it ok if i keep AutoEnforcer prefixed in headers name? there are a fair ammount of them that do

Reply:
Mmmh, header name different from class name? Sad.